Makefile

```diff
@@ -135,6 +135,9 @@ test: test-with-database
 else
 test: test-checkstyle-standalone test-with-database
 endif
+ifeq ($(CONTAINER_TEST),1)
+test: test-containers-compose
+endif
```

Comment:
Why do you want to exclude that from the "TRAVIS==true" branch?

Reply:
That was my suggestion above. As far as I can see TRAVIS is supposed to run `run-tests-within-container` which is a script that spawns a container.

Reply:
Yeah, we don't really know if this would work at all. I guess it does not matter either way.

Reply:
The above looks ok but you should also set the default of CONTAINER_TEST=1